software/speller/data_collection_platform/backend/dcp/api.py

```diff
@@ -0,0 +1,166 @@
+from functools import wraps
+from flask import Blueprint, request, current_app
+import os
+
+import dcp.mp.shared as shared
+from dcp import db
+from collections import deque
+import time
+from dcp.models.data import CollectedData
+from dcp.models.configurations import OpenBCIConfig
+
+
+bp = Blueprint('api', __name__, url_prefix='/api')
+
+
+def validate_json(*fields):
+    """Decorator to validate JSON body.
+
+    See https://flask.palletsprojects.com/en/2.0.x/patterns/viewdecorators/.
+    """
+    def decorator(f):
+        @wraps(f)
+        def decorated(*args, **kwargs):
+            if not request.is_json or not request.json:
+                return {}, 400
+
+            missing_fields = fields - request.json.keys()
+            if len(missing_fields) > 0:
+                return {'error':
+                        f'Missing fields: {", ".join(missing_fields)}'}, 400
+
+            return f(*args, **kwargs)
+        return decorated
+    return decorator
+
+
+@bp.route('/')
```

Comment:
Perhaps specify GET HTTP method?

(It's actually my first code review for NTX, please let me know if this is the type of things I should look for:))